sources/core/objects/MedbayTools.cpp

```diff
@@ -0,0 +1,69 @@
+#include "MedbayTools.h"
+
+HealthAnalyzer::HealthAnalyzer(quint32 id) : Item(id)
+{
+    SetSprite("icons/device.dmi");
+    SetState("health");
+    name = "Health Analyzer";
+}
+
+void HealthAnalyzer::Scan(IdPtr<Human> target)
+{
+    float health = target->GetHealth() / 100;
+    float suffocation_damage = target->GetSuffocationDamage() / 100;
+    float burn = target->GetBurnDamage() / 100;
+    float brute = target->GetBruteDamage() / 100;
```

Comment:
You should divide by `100.0` because otherwise it will be integer division, and it will not produce fractions.